org.gnome.baobab.json

```diff
@@ -48,8 +35,8 @@
             "builddir" : true,
             "sources" : [
                 {
-                    "type" : "git",
-                    "url" : "https://gitlab.gnome.org/GNOME/baobab.git"
+                    "type" : "archive",
+                    "url" : "https://gitlab.gnome.org/GNOME/baobab/-/archive/b5ce809b4735c20c729d99b603039de2f8718476/baobab-b5ce809b4735c20c729d99b603039de2f8718476.tar"
```

Comment:
An archive needs a sha256 for integrity checking. Also, any reason for not using https://download.gnome.org/sources/baobab/3.34/ ?

Reply:
Thank you for your reminder. The appdata.xml file of baobab 3.34 does not contain the content rating and release history required by flathub and cannot be built.

https://gitlab.gnome.org/GNOME/baobab/-/merge_requests/19

Reply:
well, you can patch the appdata file here with a source of type patch until there's a new release from upstream developers.